### ISSUE TITLE
Fine-tune pre-trained models on >3 band input

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision-fastai-plugin#20]
Fine-tuning a model pretrained on Imagenet results in a big improvement in accuracy over training from scratch, even on satellite imagery, which looks quite different than Imagenet. Unfortunately, it is not straightforward to use an Imagenet pretrained model with images with >3 bands. We would like to explore whether it is possible to take a pretrained model, and then modify it in some way so that we can get the benefits of the extra bands, while maintaining the the performance boost from transfer learning. It's not clear how to do this, or if it will work. 

The first idea we'd like to try is to load a pretrained model (UNet with Resnet18) and then modify the first convolutional layer weight array (ie. the kernel) so that it can take images with `k` channels as input. This array will have more numbers in it to account for the extra channels. It's an open question as to how to create this array. The most obvious idea is to just randomly initialize a new weight matrix of appropriate shape using the `xavier_init` routine, or whichever smart initialization technique is typically used with ResNets. Then, copy over the values from the pretrained model. 

To test this, you should use the Potsdam example (https://github.com/azavea/raster-vision-fastai-plugin/blob/master/examples/semantic_segmentation/potsdam.py 
) because it has an infrared channel which we expect to be useful for classifying vegetation, which is a class in the dataset. Using this technique, the performance should be as good, and hopefully better, than just training on or IR-R-G (infrared, red, green). https://github.com/azavea/raster-vision-fastai-plugin/blob/master/examples/semantic_segmentation/potsdam.py 

Note: #19 is a prerequisite for this issue.
